lib/cylc/gui/app_gcylc.py

```diff
@@ -2208,7 +2208,7 @@ def _popup_logview(self, task_id, task_state_summary, choice=None):
         window.set_size_request(800, 400)
         window.set_title(task_id + ": Log Files")
 
-        nsubmits = len(task_state_summary.get('job_hosts', {}))
+        nsubmits = task_state_summary.get('submit_num', 1)
```

Comment:
I've yet to have a chance to recreate & test the fix, but a quick comment: why is the default ``1`` rather than ``0`` as before? Possibly this is obvious, but I cannot see a reason.